core/sawtooth/cli/monitor.py

```diff
@@ -1,261 +0,0 @@
-# Copyright 2016 Intel Corporation
```

Comment:
monitor and stats should be kept if we intend to implement a substantially similar CLI commands.